AUTHORS

```diff
@@ -0,0 +1,3 @@
+Thomas Oster
+
+Google Inc.
```

Comment:
looks like you forgot a few of the other authors. Could you please add some reference to the git log, like:
"And many others, see the git log for details." ?